tests/test_endpoints/endpoint_data.py

```diff
@@ -229,34 +229,31 @@
             'admin2_name',
         ],
     },
-    '/api/themes/food_security': {
+    '/api/v1/food/food_security': {
```

Comment:
food-security

Reply:
At least I'm consistent! I'll fix these

Reply:
These are fixed in beb8895d196c1227d88312de0121d2733fff8678